# 单个关键点的数据
float32 x
float32 y
float32 score